prompterator/main.py

```diff
@@ -411,6 +427,20 @@ def set_up_ui_generation():
         label_visibility="collapsed",
         key=c.PROMPT_COMMENT_KEY,
     )
+    selected_model: c.ModelProperties = st.session_state.model
+    available_structured_output_settings = selected_model.supports_structured_output
+
+    structured_output_available = (
+        len(set(available_structured_output_settings) - {c.StructuredOutputImplementation.NONE})
```

Comment:
@gkaretka I mean, I know what's going on here, but you've got to admit, it does look nontrivially strange :)

Reply:
I will leave a comment there which clarifies what it does